client/src/main/proto/spine/validate.proto

```diff
@@ -41,57 +41,112 @@ import "google/protobuf/descriptor.proto";
 import "spine/annotations.proto";
 
 extend google.protobuf.MessageOptions {
+
     // The default format string for validation error message text.
+    //
+    // This option extends message types that extend `FieldOptions`
     // The number of parameters and their types are determined by the type of field options.
+    //
     string default_message = 51001;
+
+    // The option to require at least one of the fields or a combination of fields.
+    //
+    // Unlike the `required` field option which always require corresponding field,
+    // this message option allows to require alternative fields.
+    //
+    // Field names are separated using the pipe (`|`) symbol. The combination of fields is defined
+    // using the ampersand (`&`) symbol.
+    //
+    // For example:
+    //
+    // message PersonName {
+    //    option (required_field) = "given_name|honorific_prefix & family_name";
```

Comment:
@alexander-yevsyukov Let's discuss this.  I would suggest adding parenthesises. I.e.

`given_name|honorific_prefix & family_name` is different from `given_name| (honorific_prefix & family_name)`.